src/fides/api/alembic/migrations/versions/cc37edf20859_data_mig_tcf_purpose_header_and_descript.py

```diff
@@ -0,0 +1,304 @@
+                """
+                 INSERT INTO privacyexperienceconfighistory (
+                        id,
+                        component,
+                        disabled,
+                        is_default,
+                        reject_button_label,
+                        version,
+                        accept_button_label,
+                        acknowledge_button_label,
+                        banner_enabled,
+                        description, 
+                        privacy_preferences_link_label,
+                        privacy_policy_link_label,
+                        privacy_policy_url,
+                        save_button_label,
+                        title, 
+                        banner_description,
+                        banner_title, 
+                        language,
+                        name,
+                        origin,
+                        dismissable,
+                        auto_detect_language,
+                        allow_language_selection,
+                        translation_id,
+                        modal_link_label,
+                        show_layer1_notices,
+                        layer1_button_options,
+                        purpose_header
+                 )
+                 SELECT
+                    :id,
+                    component,
+                    disabled,
+                    is_default,
+                    reject_button_label,
+                    :version,
+                    accept_button_label,
+                    acknowledge_button_label,
+                    banner_enabled,
+                    :description, 
+                    privacy_preferences_link_label,
+                    privacy_policy_link_label,
+                    privacy_policy_url,
+                    save_button_label,
+                    title, 
+                    banner_description,
+                    banner_title, 
+                    language,
+                    name,
+                    origin,
+                    dismissable,
+                    auto_detect_language,
+                    allow_language_selection,
+                    translation_id,
+                    modal_link_label,
+                    show_layer1_notices,
+                    layer1_button_options,
+                    :purpose_header
+                FROM privacyexperienceconfighistory
+                WHERE id = :existing_historical_id;
+                """
+            )
+
+            bind.execute(
+                create_experience_config_history,
+                {
+                    "id": generate_record_id("pri"),
+                    "version": res["version"] + 1,
+                    "existing_historical_id": res["hist_id"],
+                    "description": updated_translation["description"],
+                    "purpose_header": updated_translation["purpose_header"],
+                },
+            )
+
+
+def downgrade():
+    pass
```

Comment:
Is there a situation where we would want/need to downgrade only `cc37` and leave `1332` in place? I feel like it's enough to say that if this was reverted both migrations would end up downgraded and the field would be deleted entirely, in which case leaving this `downgrade()` empty is fine (I think? Not super familiar with alembic). Unless we want to combine the two to make it impossible to downgrade only one.

Reply:
yes all good questions, I can probably add the downgrade on just the purpose header just in case.  Matter of preference, I like separating schema and data migrations into separate files for their distinct operations!

Reply:
Hmm I think it's okay to just rely on the column delete on the previous one, if purpose_header is cleared, there's still an ExperienceConfigHistory record with the purpose_header on it.  The downrev could delete the historical record, but it's possible other historical records were added in the interim, we can't assume there's just been one created.   I think there's less meddling if we clear the field.